filament/src/details/Camera.cpp

```diff
@@ -177,6 +187,11 @@ void UTILS_NOINLINE FCamera::setModelMatrix(const mat4& modelMatrix) noexcept {
     transformManager.setTransform(transformManager.getInstance(mEntity), modelMatrix);
 }
 
+void UTILS_NOINLINE FCamera::setEyeModelMatrix(math::mat4 const (&model)[2]) noexcept {
```

Comment:
Wondering if this should be `setEyeViewMatrix` instead (and we don't inverse()), since that's what we need? Not sure.

Reply:
We could have both?

Reply:
Actually, I think it makes sense to only offer `setEyeModelMatrix`. We don't let users set the regular camera view matrix directly either.